### DISCUSSION TITLE
Photo Resizing in V2.0.0

### DISCUSSION THREAD

**Discussion by niawag** [bpatrik/pigallery2#745] [Q&A] [answered]
Hi, I've updated PiGallery2 to its latest version and you did a great job with it, thanks a lot!
I have a question about photo resizing, I haven't changed the corresponding settings and, as I updated the app, I feel like running an indexation (and the following tasks) could be a could idea. I thought most (or all) of the photos should already be resized hence skipped but it's been running for 12h and should last for 1.5 days. I checked the tmp folder and here is what I found: 
```
-rw-r--r--  1 root root 5.8K Oct 15 22:49 P1200748.JPG_240q80cs.webp
-rw-r--r--  1 root root 110K Jan 14  2023 P1200749.JPG_1080q80.webp
-rw-r--r--  1 root root 115K Oct 16 12:26 P1200749.JPG_1080q80cs.webp
```
The first line is the thumbnail, the second is the resized picture from january and the last is the resized picture from today. It seems there was a change in  the conversion settings, is this normal? Should I remove all the resized photos from january ? Would it be possible to have a setting to skip already resized photos (even if the new resize algo is better) ?

**Comment by bpatrik** [comment, marked as answer]
This is because the def. photo resizing algorithm changed. You can disable it:

![Screenshot_20231017-085004.png](https://github.com/bpatrik/pigallery2/assets/7512182/d9a4c590-f844-405c-aedb-e5c5f5bc220e)

**Reply by bpatrik** [reply]
Also if you run temp folder cleaning it will remove those thumbnail photos that are "not valid" with the current settings.

**Comment by niawag** [comment]
Thanks a lot, the process is complete now so I'll keep the new new resized pics with chroma subsampling. Temp folder cleaning was also automatically performed and all is OK.
I had'nt found this setting because I was'nt lookin in thumbnail generation  but in Photo Resizing.

**Comment by bpatrik** [comment]
Adding a reference to this discussion to the release notes.
